wingtips-zipkin2/src/test/java/com/nike/wingtips/zipkin2/util/WingtipsToZipkinSpanConverterDefaultImplTest.java

```diff
@@ -6,6 +6,7 @@
 import com.tngtech.java.junit.dataprovider.DataProvider;
 import com.tngtech.java.junit.dataprovider.DataProviderRunner;
 
+import org.assertj.core.api.Condition;
 import org.junit.Test;
```

Comment:
The code coverage numbers for this diff are pretty low - you must be missing some of the branches. Please add test cases to cover everything. I'm guessing most of them can be covered by simply adding new cases to the `@DataProvider`s. It should be pretty easy to hit 100% coverage for this diff.